quality_control_stock/models/stock_picking.py

```diff
@@ -46,9 +48,20 @@ def do_transfer(self):
                           'qc.trigger.product_line']:
                 partner = (self.partner_id
                            if qc_trigger.partner_selectable else False)
-                trigger_lines = trigger_lines.union(
-                    self.env[model].get_trigger_line_for_product(
-                        qc_trigger, operation.product_id, partner=partner))
+                if not operation.product_id:
+                    dict_product = package_obj._get_all_products_quantities(
```

Comment:
Isn't better to just retrieve products with a `mapped` operation over the one2many field of the details of the package? You don't need quantities